qa/workunits/rados/test_rados_tool.sh

```diff
@@ -52,18 +52,30 @@ run() {
 DNAME="`dirname $0`"
 DNAME="`readlink -f $DNAME`"
 RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
-if ! test -f $RADOS_TOOL ; then
-    RADOS_TOOL=$(which rados)
+if [ ! -f "$RADOS_TOOL" ] ; then
```

Comment:
@tchaikov 
We can/could do that.
The readlink stuff makes sense to me, but getting it out will make the code much more legible. But it emphasizes the need to be executed from a parent script setting these vars. I'll try and patch as suggested 

I sort of wonder why we do  `$(which rados)` anyways. it will run rados from an installed ceph installation, and I would not really like to start using that whilest testing.